docs/core/admin/limits.md

```diff
@@ -3,16 +3,43 @@ layout: doc
 title: Limits
 ---
 
-# Dovecot Built-in Limits
+# Dovecot Limits
 
-Dovecot contains various built-in resource limits designed to prevent
-denial of service situations. This page lists those limits.
+Dovecot contains various configurable and built-in resource limits designed to
+prevent denial of service situations. This page lists those limits.
 
 ::: todo
 This list is currently incomplete.
 :::
 
-## Message Headers
+## Configurable Limits
+
+Storage size limits:
+
+ * [[link,quota_root]
+ * [[setting,quota_max_mail_size]
```

Comment:
It would also be possible, at least for the settings, to add a tag in data/settings.js (e.g. "storage-size-limits"), and then auto-generate a mini-settings list with `<SettingsComponent tag="storage-size-limits />`.

This would make it trivial to add additional settings only in Pro, by just including that tag in the settings.js file.

However, the links would have to remain being handled separately.

I leave it up to you whether this is a good idea or not.

Reply:
I dropped a bit of the title hierarchy, since it looked so bad.